Watch Extension/State Management/CommunicationErrorReducer.swift

```diff
@@ -6,11 +6,13 @@ enum CommunicationErrorFyiDialogAction {
 }
 
 func communicationErrorFyiDialogReducer(showCommunicationFyiDialog: inout Bool,
-                                        action: CommunicationErrorFyiDialogAction) {
+                                        action: CommunicationErrorFyiDialogAction) -> [Effect<CommunicationErrorFyiDialogAction>] {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/995ebb3b78eb3cd87dba718e07e1f295f49795f1/Watch Extension/State Management/CommunicationErrorReducer.swift#L9"></span>Line should be 120 characters or less: currently 131 characters<br />
<code>line_length</code> <code>CommunicationErrorReducer.swift:9</code></td>
    </tr>
  </tbody>
</table>